### DISCUSSION TITLE
Failed to fetch 7TV

### DISCUSSION THREAD

**Discussion by moadAlami** [Chatterino/chatterino2#3344] [Q&A] [answered]
Whenever I start chatterino I get this error
`Failed to fetch 7TV channel emotes. (unknown error)`
I'm using the 7.3.4 build on Windows 10. The same build is working fine on another pc tested both on windows 10 and archlinux.

**Comment by Felanbird** [comment, marked as answer]
We do not support 7TV on the main branch -> https://github.com/SevenTV/chatterino7/discussions/new

**Reply by moadAlami** [reply]
My bad, I'll redirect it to chatterino7.
